### PR TITLE
Format 100.00 as 100% instead of 100.00%.

### PR DESCRIPTION
**Pull request by aureooms**

### CODE REVIEW

File: src/helpers/calculate-uptime.ts [hunk 1]
```diff
@@ -57,5 +57,6 @@ export const getUptimePercentForSite = async (slug: string): Promise<string> =>
   const downtimeSeconds = await getDowntimeSecondsForSite(slug);
 
   // Return a percentage string
-  return `${Math.max(0, 100 - (downtimeSeconds / totalSeconds) * 100).toFixed(2)}%`;
+  const percentage = Math.max(0, 100 - (downtimeSeconds / totalSeconds) * 100).toFixed(2);
+  return percentage === "100.00" ? "100%" : `${percentage}%`;
```
Comment (aureooms) [inline src/helpers/calculate-uptime.ts:61]: Another way would be to use `parseFloat` like here: https://github.com/aureooms-contrib/graphs/blob/4c2348f4b9a2e37b849bfedc7e9178c7d5bdd953/src/index.ts#L42

**Comment by AnandChowdhary** [comment]
Any specific reason you want to format 100.00% to 100%? I would miss the consistency, since we have 98.97% (all numbers are 2 decimal places), so why keep 100% as a special case? 😄

**Comment by aureooms** [comment]
> Any specific reason you want to format 100.00% to 100%? I would miss the consistency, since we have 98.97% (all numbers are 2 decimal places), so why keep 100% as a special case? smile

Because you cannot have `100.01%`. Other than that the choice is arbitrary.

**Comment by aureooms** [comment]
Also an uptime of (exactly) 100% is unlikely. I do not know what is the best way to approach uptimes of say 99.995%. Maybe two digits is enough for most users of upptime. Maybe time will tell.

**Pull request closed without merging**